roles/openshift_aws/defaults/main.yml

```diff
@@ -226,7 +228,7 @@ openshift_aws_ami_map:
   infra: "{{ openshift_aws_ami }}"
   compute: "{{ openshift_aws_ami }}"
 
-openshift_aws_master_group:
+openshift_aws_node_groups:
```

Comment:
I don't see why openshift_aws_master_group is combined with openshift_aws_node_groups.  They appear to be completely separate.

Reply:
This is OCP node group data. An update would mean getting into the OpenShift install code and changing.  Cluster Operator would also need to be updated.  That is something that we should not take on here.

Reply:
I don't understand what you mean.

In this pr, openshift_aws_node_groups has been combined with openshift_aws_master_group.  I don't see a reason for doing so in this PR, better to leave these two things separate instead of combining into one big dictionary.